GNUmakefile

```diff
@@ -46,7 +46,7 @@ attach:
 
 acceptance: fmtcheck
 	export TF_ACC=true && \
-	go test -ldflags="-X '${PKG_VERSION_PATH}.Version=${NEXT_VERSION}-test'" -v -parallel 20 -timeout 20m ./pkg/...
+	go test -ldflags="-X '${PKG_VERSION_PATH}.Version=${NEXT_VERSION}'" -v -parallel 20 -timeout 20m ./pkg/...
```

Comment:
why did you remove `-test`? It might create a noise in BI data.

Reply:
Ok. Fair enough.

Reply:
I've noticed same in RT provider and in projects. A bit too late, but we need to remove it as well please.